CMakeLists.txt

```diff
@@ -20,13 +21,20 @@ if(NOT DEFINED OPENCV_LIBS)
 endif()
 
 if(NOT DEFINED CPPFILESYSTEM)
-    set(CPPFILESYSTEM "stdc++fs")
+	set(CPPFILESYSTEM "stdc++fs")
+endif()
+
+if(DEFINED USE_GLFW)
+	set(libcimbar_UI_LIB "glfw" "GL")
+	add_definitions("-DLIBCIMBAR_USE_GLFW=${USE_GLFW}")
+else()
+	set(libcimbar_UI_LIB "opencv_highgui")
 endif()
 
 set (PROJECTS
 	src/exe/cimbar
-	src/exe/cimbar_scan
-	src/exe/cimbar_video
+	src/exe/cimbar_extract
```

Comment:
Also renamed `cimbar_send` to `cimbar_extract`, to better reflect what you might use it for.